tests/test_pytest/test_plugin_schema/bad/bases/valid_identifier.yaml

```diff
@@ -1,12 +1,13 @@
----
-1
----
-["a", "b"]
----
-class
----
-else
----
-bl@h
----
-abi-infrared
+# ---
+# 1
```

Comment:
You commented out these tests since the validator is disabled ?

Reply:
Exactly. These fail if they remain.